src/k-provider/ovp/provider.ts

```diff
@@ -349,6 +350,7 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     sourcesObject.type = mediaEntry.type;
     sourcesObject.dvr = !!mediaEntry.dvrStatus;
     sourcesObject.poster = mediaEntry.poster;
+    sourcesObject.rawThumbnailUrl = mediaEntry.rawThumbnailUrl;
```

Comment:
can we return the sources object from provider with rawThumbnailUrl only, and add the poster field in the player ? what will that affect ?

Reply:
i am not sure, there is a large usage of poster in providers repo- i don't feel comfortable with doing that.

Reply:
fair enough